sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_autoinstrumentation/configurator.py

```diff
@@ -26,6 +33,17 @@ def _configure(self, **kwargs):
         if not _is_attach_enabled():
             warn(_PREVIEW_ENTRY_POINT_WARNING)
         try:
+            if environ.get(OTEL_TRACES_EXPORTER, "").lower().strip() != "none":
+                kwargs.setdefault("trace_exporter_names", ["azure-monitor-opentelemetry-exporter"])
```

Comment:
Maybe want to move these names into `constants`

Reply:
Can do.

Reply:
Done.